diff -urN SDL_mixer.orig/debian/control SDL_mixer/debian/control
--- SDL_mixer.orig/debian/control	2013-04-25 21:20:44.511744288 -0400
+++ SDL_mixer/debian/control	2013-04-25 21:23:37.153573885 -0400
@@ -12,7 +12,7 @@
                libfluidsynth-dev,
                libmikmod2-dev (>= 3.1.11),
                libogg-dev (>= 1.2.0),
-               libsdl2-dev (>= 2.0.0),
+               libsdl2-dev (>= 2.0.0~),
                libtool,
                libvorbis-dev (>= 1.2.0)
 Homepage: http://www.libsdl.org/projects/SDL_mixer/
@@ -37,7 +37,7 @@
 Depends: ${misc:Depends},
          libsdl2-mixer (= ${binary:Version}),
          libc6-dev,
-         libsdl2-dev (>= 2.0.0)
+         libsdl2-dev (>= 2.0.0~)
 Description: Mixer library for Simple DirectMedia Layer 1.2, development files
  SDL Mixer is a sample multi-channel audio mixer library.  It supports 4
  channels of 16 bit stereo audio, plus a single channel of music, mixed by
