diff -urN SDL_image.orig/debian/control SDL_image/debian/control
--- SDL_image.orig/debian/control	2013-04-25 22:24:19.207787062 -0400
+++ SDL_image/debian/control	2013-04-26 21:20:55.383020119 -0400
@@ -1,16 +1,17 @@
 Source: libsdl2-image
 Section: libs
 Priority: optional
-Maintainer: Debian SDL packages maintainers <pkg-sdl-maintainers@lists.alioth.debian.org>
+Maintainer: Michael Imamura <zoogie@lugatgt.org>
 Uploaders: Barry deFreese <bdefreese@debian.org>,
            Aurelien Jarno <aurel32@debian.org>,
            Felix Geyer <debfx-pkg@fobos.de>,
-           Sam Lantinga <slouken@libsdl.org>
+           Sam Lantinga <slouken@libsdl.org>,
+           Michael Imamura <zoogie@lugatgt.org>
 Standards-Version: 3.9.2
 Build-Depends: debhelper (>= 8.9.0~),
                dh-autoreconf,
                dpkg-dev (>= 1.16.1~),
-               libsdl2-dev (>= 2.0.0),
+               libsdl2-dev (>= 2.0.0~),
                libjpeg-dev,
                libpng-dev,
                libtiff4-dev,
@@ -24,6 +25,8 @@
 Depends: ${misc:Depends},
          ${shlibs:Depends}
 Description: Image loading library for Simple DirectMedia Layer 2.0
+ *** Unofficial package from ppa:zoogie/sdl2-snapshots ***
+ .
  This is a simple library to load images of various formats as SDL surfaces.
  This library currently supports BMP, PPM, PCX, GIF, JPEG, PNG, TIFF, and XPM
  formats.
@@ -37,8 +40,10 @@
 Depends: ${misc:Depends},
          libsdl2-image (= ${binary:Version}),
          libc6-dev,
-         libsdl2-dev (>= 2.0.0)
+         libsdl2-dev (>= 2.0.0~)
 Description: development files for SDL 2.0 image loading library
+ *** Unofficial package from ppa:zoogie/sdl2-snapshots ***
+ .
  This is a simple library to load images of various formats as SDL surfaces.
  This library currently supports BMP, PPM, PCX, GIF, JPEG, PNG, TIFF, and XPM
  formats.
